### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

**Comment by nedbat** [comment]
You need to run edx-lint to write the files in the root of the directory. There are instructions in the file about what to do.  Instead of editing those files, update write.py (as you have done), then run `edx-lint write .editorconfig` (etc) to update the .editorconfig file.

**Comment by nedbat** [comment]
Now this PR doesn’t update the .editorconfig file itself in edx-lint. Is that intentional? I can understand if so: get a new edx-lint, publish it, and then use it on everything, including itself?

**Comment by sarina** [comment]
@nedbat - the build is now failing due to [two instances of R201](https://github.com/openedx/edx-lint/runs/8235130294?check_suite_focus=true), R0201: Method could be a function (no-self-use).

Can I ignore these? Is this a helpful warning for the project?

**Comment by nedbat** [comment]
Yes, we can silence those violations, but it's not easy.  See #283.

**Comment by sarina** [comment]
That worked, thanks @nedbat 

Could I get a re-review from you or @feanil ?

File: edx_lint/pylint/module_trace.py
Comment (feanil) [inline edx_lint/pylint/module_trace.py:44]: Is there a reason this pointless statement is being left here? Can that reason be left in a comment next to this?

File: test/test_cmd.py
Comment (feanil) [inline test/test_cmd.py:45]: Same as the comment above, if this is not useless, A comment would help and if it is useless, then removing the line would be good.

**Review by feanil — CHANGES_REQUESTED** [review]

File: edx_lint/pylint/module_trace.py
Comment (sarina) [reply edx_lint/pylint/module_trace.py:44]: @nedbat did this to silence the R201 warning, see https://github.com/openedx/edx-lint/pull/283

**Review by feanil — COMMENTED** [review]
Weird that we didn't remove the lines but let's unblock this for now and we can deal with that as a follow up.

**Review by feanil — APPROVED** [review]

**Comment by sarina** [comment]
Thanks! I'll create a tag for `5.2.5` now, hoping that triggers a release.

**Pull request merged**
